.github/workflows/beam_PreCommit_SQL_Java11.yml

```diff
@@ -38,7 +38,7 @@ concurrency:
 permissions:
     actions: write
     pull-requests: read
-    checks: read
+    checks: write
```

Comment:
Am I understand correctly that the write permission is now required for the SpotBugs plugin?

Reply:
checks: write (allow to add test result to workflow summary page)